.gitignore

```diff
@@ -11,5 +11,6 @@ packaged
 coverage
 public/service-worker.js
 private-key.json
-
+.yarn-error.log
```

Comment:
Would you mind updating the .gitignore to just ignore all `.log` files?

```
*.log
```

Reply:
wow cool, i mind only ignore `yarn-error.log` but ignore all `.log` really cool